### PR TITLE
doc: improve docs for std::io::Seek::rewind method

### PR DESCRIPTION
**Pull request by poliorcetics**

### CODE REVIEW

**Comment by vacuus** [comment]
Should it really keep the line that it's equivalent to the other function call, then?

File: library/std/src/io/mod.rs [hunk 1]
```diff
@@ -1770,6 +1770,9 @@ pub trait Seek {
     /// Rewind to the beginning of a stream.
     ///
     /// This is a convenience method, equivalent to `seek(SeekFrom::Start(0))`.
+    /// Note that unlike `seek(SeekFrom::Start(0))`, this method does not return
+    /// the old offset and cannot replace assignements of the form:
+    /// `let old_offset = buf.seek(SeekFrom::Start(0))?;`.
```
Comment (the8472) [inline library/std/src/io/mod.rs:1775]: `seek` doesn't return the old position unless you do `SeekFrom::Current(0)`

Comment (m-ou-se) [reply library/std/src/io/mod.rs:1775]: See also [the documentation of seek](https://doc.rust-lang.org/stable/std/io/trait.Seek.html#tymethod.seek):

> If the seek operation completed successfully, this method returns the **new** position from the start of the stream.

**Comment by Dylan-DPC** [comment]
@poliorcetics any updates on this?

**Comment by Dylan-DPC** [comment]
Closing this as inactive. Feel free to reöpen this pr or create a new pr if you get the time to work on this. Thanks

**Comment by tgross35** [comment]
@Dylan-DPC based on your last comment did you mean to close this? It is still open

**Comment by Dylan-DPC** [comment]
Ye sorry, thanks :)

**Pull request closed without merging**